src/canister/individual_user_template/src/data_model/mod.rs

```diff
@@ -163,7 +164,8 @@ impl Default for CanisterData {
             device_identities: Vec::new(),
             ml_feed_cache: Vec::new(),
             cdao_canisters: Vec::new(),
-            token_roots: _default_token_list()
+            token_roots: _default_token_list(),
+            user_propensity: 0.0,
```

Comment:
I believe this is related to ML data. Could we have a custom struct and store this field inside that?